packages/connect/src/device/Device.ts

```diff
@@ -116,13 +116,17 @@ type DeviceParams = {
 
 export class Device extends TypedEmitter<DeviceEvents> {
     public readonly transport: Transport;
-    public readonly protocol: TransportProtocol;
     public readonly transportPath;
     public readonly bluetoothProps;
     private thp: protocolThp.ThpState | undefined;
     private readonly transportDescriptorType;
     private sessionAcquired: Session | null;
 
+    private _protocol: TransportProtocol;
+    public get protocol() {
+        return this._protocol;
+    }
```

Comment:
btw is this better then simply accessing public readonly protocol?

Reply:
idk. [i had it as public](https://github.com/trezor/trezor-suite/pull/19122#discussion_r2104157001) then i decided to adjust to the rest of the code